psydac/api/ast/utilities.py

```diff
@@ -183,10 +181,43 @@ def _get_name(atom):
 def compute_atoms_expr_field(atomic_exprs, indices_quad,
```

Comment:
@saidctb Given that the return values of this function have been changed, I think it is time to add a docstring to it with a brief explanation of what it does, and a description of the input parameters and return values. It should look like
```
"""
This function computes [etc etc...]

Parameters
----------
atomic_exprs : <type>
    <description>

indices_quad : <type>
    <description>

idxs : <type>
    <description>

basis : <type>
    <description>

test_function : <type>
    <description>

mapping : <type>
    <description>

Returns
-------
inits : <type>
    <description>

updates : <type>
    <description>

map_stmts : <type>
    <description>

atomic_exprs: <type>
    <description>
"""
```
Feel free to use this as a template. It should only take 5 minutes to add this, and it will be greatly useful for all future maintenance.